docs/synapse/devopsguide.rst

```diff
@@ -981,6 +981,123 @@ this to add a user named ``foouser`` with the uid 1234::
     Successfully built 21a12f395462
     Successfully tagged vertexproject/synapse-aha:v2.113.0-foouser
 
+Configure Custom CA Certificates for HTTP Requests
+--------------------------------------------------
+
+The Cortex can be configured to use additional CA certificates when making HTTP
+requests via Storm. To do this, you need to provide the certificates (in DER or
+PEM format) to the Cortex in a directory on disk, and then configure the Cortex
+to look up that directory via the ``tls:ca:dir`` configuration option.
+
+The following Compose file shows an example using this option.
+
+::
+
+    services:
+      00.cortex:
+        user: "999"
+        image: vertexproject/synapse-cortex:v2.x.x
+        network_mode: host
+        restart: unless-stopped
+        volumes:
+            - ./storage:/vertex/storage
+        environment:
+            SYN_CORTEX_TLS_CA_DIR: /vertex/storage/tls-ca-certs
+
+.. note::
+
+   CA certificates provided via the ``tls:ca:dir`` configuration option MUST
+   include the full chain of certificates all the way up to the root. Providing
+   only partial CA chains may result in verification failures.
+
+Configure Custom CA Certificates with Kubernetes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To configure custom CA certificates with kubernetes, do the following:
+
+#. Create a config map from your CA certificate files.
+
+    The example has a pre-created root CA and an intermediate CA certificate in PEM format::
+
+        $ ls -l ./cas
+        total 8
+        -rw-rw-r-- 1 user user 1708 Feb 14 19:19 intermediate.crt
+        -rw-rw-r-- 1 user user 1696 Feb 14 19:19 root.crt
+
+        $ kubectl create configmap zscalar-ca-certs --from-file ./cas
```

Comment:
Lets call this configmap something generic such as `tls-ca-certs`.

Reply:
Thanks, I meant to update that and missed it